app/templates/components/filter-option.hbs

```diff
@@ -0,0 +1,26 @@
+<form class="ui large form">
+  <div class="fields">
```

Comment:
Instead of this use semantic action input. (Ref: https://semantic-ui.com/elements/input.html)

input + input + semantic-dropdown + action